.github/workflows/create-release.yaml

```diff
@@ -0,0 +1,38 @@
+name: Create Release
```

Comment:
this release build runs on every push but only releases if the version number has changed